<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>文本转语音</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.PNG</normaloff>icon.PNG</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainWindow{background:white}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{background:white}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{background:white}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
         <item>
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_select_dir">
             <property name="styleSheet">
              <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 16px;
    color: #ffffff;
    background-color: rgb(206, 175, 36);
    border-style: solid;
    border-width: 2px;
    border-color: gray;
    border: 1px solid gray;
}

QPushButton:hover{ background-color: #b4bd3a;}
QPushButton:pressed{  background-color:  #a5ad35;}</string>
             </property>
             <property name="text">
              <string>选择路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonlisten">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 16px;
    color: #ffffff;
    background-color: rgb(206, 175, 36);
    border-style: solid;
    border-width: 2px;
    border-color: gray;
    border: 1px solid gray;
}

QPushButton:hover{ background-color: #b4bd3a;}
QPushButton:pressed{  background-color:  #a5ad35;}</string>
           </property>
           <property name="text">
            <string>试听声音</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_start">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 16px;
    color: #ffffff;
    background-color: rgb(206, 175, 36);
    border-style: solid;
    border-width: 2px;
    border-color: gray;
    border: 1px solid gray;
}

QPushButton:hover{ background-color: #b4bd3a;}
QPushButton:pressed{  background-color:  #a5ad35;}</string>
           </property>
           <property name="text">
            <string>开始转换</string>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionreadme"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionreadme">
   <property name="text">
    <string>Readme</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
